test/kixi/hecuba/weather_test.clj

```diff
@@ -2,6 +2,4 @@
   (:require [clojure.test :refer :all]
             [kixi.hecuba.weather :refer :all]))
 
-(deftest a-test
-  (testing "FIXME, I fail."
-    (is (= 0 1))))
+
```

Comment:
Here's the test I wrote on Friday: https://github.com/MastodonC/kixi.hecuba.weather/blob/feature/upload-daily-measurements/test/kixi/hecuba/weather/metoffice_api_test.clj.
Should I add to it?